TESTS/host_tests/sdk_host_tests.py

```diff
@@ -96,13 +97,27 @@ def setup(self):
         self.register_callback('fail_test', self._callback_fail_test)
         
         # Setup API config
-        api_key_val = os.environ.get("MBED_CLOUD_API_KEY")            
+        try:
+            result = subprocess.check_output(["mbed", "config", "-G", "MBED_CLOUD_SDK_CONFIG"], \
```

Comment:
What is this, and why do we have it here? This seems way too complicated.

Reply:
I've just push changes to simplify this and reuse the `mbed config` option :)
Can you review?